platemate/management/models/turk.py

```diff
@@ -91,17 +91,24 @@ def work_time(self):
     # Returns True if all responses are valid
     @property
     def valid(self):
-        for r in self.responses.all():
+        responses = self.responses.all()
+        if not responses:
```

Comment:
So if a QuerySet has no results it evaluates to False? Just checking. I'm sure you've tested this...